addons/mass_mailing_sms/wizard/sms_composer.py

```diff
@@ -45,7 +45,7 @@ def _prepare_mass_sms_trace_values(self, record, sms_values):
             trace_values['ignored'] = fields.Datetime.now()
         else:
             if self.mass_sms_allow_unsubscribe:
-                sms_values['body'] = '%s %s' % ((sms_values['body'] or ''), self._get_unsubscribe_url(record.id, trace_code, sms_values['number']))
+                sms_values['body'] = _('%s\nSTOP SMS : %s') % ((sms_values['body'] or ''), self._get_unsubscribe_url(record.id, trace_code, sms_values['number']))
```

Comment:
Isn't it better
``
%s\n%s % (sms_values['body'] or '', _('STOP SMS: %s') % self._get_unsubscribe_url(record.id, trace_code, sms_values['number']))
``
?

Reply:
That way we correctly concatenate the body and then a translated part.